recipes/ndarray_listener/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ndarray_listener" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "95d637109bbb647ad54fb57b982f9881d4501394395e82b0e1f484657ac29ec1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+  run:
+    - python
+    - numpy
+    - pytest
```

Comment:
But I need pytest during runtime. My packages can be tested also in runtime, by the final user...